experiments/chaos/openebs_target_network_loss/test.yml

```diff
@@ -219,7 +218,10 @@
                 containername: "{{ pv.stdout }}-ctrl-con"
               when: 
                 - stg_prov == 'openebs.io/provisioner-iscsi'
+          when:
```

Comment:
we can use only one when condition here, like below
when:
    - stg_prov
    -stg_engine